lib/rss/dublincore.rb

```diff
@@ -81,7 +81,7 @@ module DublinCoreModel
     }
 
     ELEMENT_NAME_INFOS = DublinCoreModel::TEXT_ELEMENTS.to_a
-    DublinCoreModel::DATE_ELEMENTS.each do |name, |
+    DublinCoreModel::DATE_ELEMENTS.each do |name,|
```

Comment:
```suggestion
    DublinCoreModel::DATE_ELEMENTS.each do |name, |
```

Not in `Style/StringLiterals`.